google-cloud-clients/google-cloud-spanner/pom.xml

```diff
@@ -27,6 +28,7 @@
         <version>2.12.4</version>
         <configuration>
           <excludedGroups>com.google.cloud.spanner.IntegrationTest</excludedGroups>
+          <skipTests>${skipUnitTests}</skipTests>
```

Comment:
is there a separate unit test rule too ?

Reply:
I'm not following. This _is_ the unit test rule. surefire plugin is for unit tests.